examples/logger.py

```diff
@@ -85,11 +126,19 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame = ConfirmClearingFrame()
         self.confirmFrame.confirmed.connect(self.clearLog)
+        self.handler = LoggingHandler(self.addLog)
```

Comment:
Why did you make `handler` as an attribute of `LoggerApp`?
Is it necessary?

Reply:
No as far as I check, it doesn't matter whether setting it as local or class variable.

Reply:
Maybe setting it as an attribute would be good because it allows users to access the handler object.